containers/groovy/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: "3.4"
+
+services:
+  grpc-api:
+    image: ghcr.io/deephaven/grpc-api:${VERSION:-latest}
+    expose:
+      - '8080'
+    volumes:
+      - ./docker/core/data:/data
```

Comment:
The `./docker/core/data` path seems funny for this use case.  Kind of Rube Goldberg.  Should we be using `./data` instead?